packages/back-end/src/controllers/admin.ts

```diff
@@ -96,3 +231,131 @@ export async function putOrganization(
     status: 200,
   });
 }
+export async function getMembers(
+  req: AuthRequest<never, never, { page?: string; search?: string }>,
+  res: Response
+) {
+  if (!req.superAdmin) {
+    return res.status(403).json({
+      status: 403,
+      message: "Only superAdmins can get all members",
+    });
+  }
+
+  const { page, search } = req.query;
+
+  const organizationInfo: Record<string, object> = {};
+  const allUsers = await getAllUsersFiltered(parseInt(page ?? "1"), search);
+  if (allUsers?.length > 0) {
+    for await (const user of allUsers) {
+      const orgs = await findOrganizationsByMemberId(user.id);
```

Comment:
This has the potential to be inefficient. I believe you are loading 50 members a page and hence this would make 50 db requests. It is probably worth it to make a new query that finds all organizations by a list of members and then create your array from the result of that single query.

Reply:
done